phpdotnet/phd/Package/PHP/XHTML.php

```diff
@@ -880,7 +880,7 @@ public function format_classsynopsis_oo_name_text($value, $tag) {
     }
 
     public function format_classname_text($value, $tag) {
-        if (($filename = $this->getClassnameLink(strtolower($value))) !== null && $this->cchunk["class_name_ref"] !== strtolower($value)) {
+        if (($filename = $this->getClassnameLink($this->normalizeFQN($value))) !== null && $this->cchunk["class_name_ref"] !== strtolower($value)) {
```

Comment:
This is somewhat unrelated, but in a future version, first test if `getClassnameLink()` with fails with raw string, and then test again with plain `\trim()`. In this case, generate the link as usual but generate an warning, as this a source error (unexpected whitespace around type/constante/etc tags).